src/components/RunMenu.vue

```diff
@@ -5,7 +5,7 @@
         ref="runButton"
         class="run-menu__run-button"
         icon-append="PlayIcon"
-        :disabled="deployment.deprecated"
+        :disabled="deployment.deprecated || deployment.disabled"
```

Comment:
I think same question here, though maybe it's ok if this is disabled; is this an API restriction as well?

Reply:
Correct -- attempting to create a flow run on a disabled deployment returns a 409